templates/web/base/maps/noscript_map_wms.html

```diff
@@ -0,0 +1,32 @@
+<div class="noscript square-map__outer">
```

Comment:
This template is very similar to the WMTS one; could the pin location calculations be factored out?